phonenumber_field/phonenumber.py

```diff
@@ -87,7 +87,8 @@ def to_python(value):
             phone_number = PhoneNumber(raw_input=value)
     elif (isinstance(value, phonenumbers.phonenumber.PhoneNumber) and
           not isinstance(value, PhoneNumber)):
-        phone_number = PhoneNumber(value)
+        phone_number = PhoneNumber()
```

Comment:
Because constructor of ```PhoneNumber``` class (which you call by ```PhoneNumber(value)``` on line 90), has as the first positional argument ```country_code```. It is the reason why is not possible simply cast ```value```. So I create "empty" ```PhoneNumber``` instance instead and then, on next line (91), set it to the same ```value``` by ```merge_from```, which correctly handle any type of PhoneNumber.